adaptors/fhir-4.md

```diff
@@ -0,0 +1,710 @@
+---
+title: FHIR r4 Adaptor
```

Comment:
Do we _want_ to mention the option for the FHIR implementation guide-specific adaptors? Or no, not here
https://github.com/OpenFn/adaptors/wiki/Generating-Fhir-Adaptors
https://docs.openfn.org/documentation/get-started/standards#fhir-adaptors

Reply:
Not yet because it's gonna be broken when we release :upside_down_face: I'll need to do a bunch of refactoring to get that stuff working again. Mostly because of the change to typescript

Reply:
@josephjclark and no way to version-lock what was there before? Otherwise, pls own updating those 2 links above accordingly or lmk if I can help